ui/app/templates/components/license-banners.hbs

```diff
@@ -1,24 +1,23 @@
 {{#if this.licenseExpired}}
   <div class="license-banner-wrapper" data-test-license-banner data-test-license-banner-expired>
     <AlertBanner
```

Comment:
no longer using the param bannerType which was used on a data-test selector but not being used by any tests.